py/__init__.py

```diff
@@ -11,10 +11,10 @@ def py_types_encode_decode(t):
 _pattern2 = re.compile("([a-z0-9])([A-Z])")
 
 
-def py_param_name(type_name):
+def py_param_name(type_name, check_for_keywords=True):
```

Comment:
why do we need this boolean?

Reply:
await is a keyword in python, and without it, we were generating a codec file named `count_down_latch__await_codec.py` instead of `count_down_latch_await_codec.py` (double vs single underscore before await)